toqito/channel_metrics/fidelity_of_separability.py

```diff
@@ -121,7 +121,7 @@ def fidelity_of_separability(
     :raises ValueError: the input state is a mixed state.
     :return: Optimized value of the SDP when maximized over a set of linear
         operators subject to some constraints.
-    """
+    """ # noqa: E501
```

Comment:
Temporarily disabled lin `length > 120` for this docstring. Make changes to `math` before marking this PR ready for review.